packages/react/counter/src/Counter.tsx

```diff
@@ -118,18 +136,19 @@ export function Counter(props: CounterProps) {
     async function setup() {
       if (isStatic && value) {
         setDataValue(value)
-        return
       }
 
-      const fetchedValue = await fetchData()
+      if (!isStatic) {
+        const fetchedValue = await fetchData()
 
-      if (!fetchedValue) {
-        setHasError(true)
-        console.error(`QueryError: Your metric ${query?.metric} returned undefined or a \`null\` value.`)
-        return
-      }
+        if (typeof fetchedValue === 'undefined') {
```

Comment:
@felipecadavid how about this instead?
```suggestion
        if (fetchedValue === undefined) {
```

Reply:
Yes, that should work the same way, I'm refactoring this